app/services/nytimes_service.rb

```diff
@@ -47,19 +46,24 @@ def get_most_recent_obituaries
   end
 
   def get_advanced_obituaries(name, year)
-
     response = conn.get do |req|
       req.params['fq'] = "section_name:(obituaries) AND headline:(#{name}) AND pub_year:(#{year})"
     end
     JSON.parse(response.body, symbolize_names: true)
   end
 
+  def get_obituaries_by_year(year)
+    response = conn.get do |req|
+      req.params['fq'] = "section_name:(obituaries) AND pub_year:(#{year})"
```

Comment:
This is the only new method I made. It only requests the year with no name

Reply:
![](https://media2.giphy.com/media/nXxOjZrbnbRxS/giphy.gif?cid=5a38a5a29c4zlx1cjse0ygslo3tdg56we95430t4mrb8xffe&rid=giphy.gif)